src/environment.ts

```diff
@@ -232,7 +232,7 @@ export class Environment {
           })`;
         }
       } else {
-        if (name === "unescape") {
+        if (name === "unescape" || name === "safe") {
```

Comment:
I think only `safe` filter should be taken into account here and treat `unescape` as any other filter, without any special behavior.